server/jwt.go

```diff
@@ -168,7 +167,7 @@ func removeUserCookie(c echo.Context) {
 func JWTMiddleware(l *zap.Logger, p api.PrincipalService, next echo.HandlerFunc, mode string, secret string) echo.HandlerFunc {
 	return func(c echo.Context) error {
 		// Skips auth, actuator, plan
-		if strings.HasPrefix(c.Path(), "/api/auth") || strings.HasPrefix(c.Path(), "/api/actuator") || strings.HasPrefix(c.Path(), "/api/plan") {
+		if common.HasPrefixes(c.Path(), "/api/auth", "/api/plan") {
```

Comment:
Missing /api/actuator ?

Reply:
Good catch. BTW, do you have a good name for these paths, maybe we can have a variable for them? What about `anonyPaths` or `noAuthPaths`?

Reply:
noAuthPaths SGTM. We can do it in a separate PR